src/constants.ts

```diff
@@ -11,10 +11,8 @@ import {
 
 export const MIN_LEVEL: number = 1;
 export const MAX_LEVEL: number = 275;
-export const MIN_TIMES_ENLIGHTENDED: number = 0;
```

Comment:
This was not used anywhere, other than an import that wasn't used. We _could_ use it in the range input, but things like 0 or 1 seem like more obvious literals than maximums. Introducing minimum constants for other ranges seems tedious and low-value add (given the song and dance needed to get it available to the template)